modules/orbitsoftBidAdapter.js

```diff
@@ -0,0 +1,232 @@
+      utils.logMessage('JSONP callback function called for ad ID: ' + id);
+      // @endif
+
+      let bid = [];
+      if (jptResponseObj.cpm && jptResponseObj.cpm !== 0) {
+        // Store bid response
+        responseCPM = jptResponseObj.cpm;
+        // Bid status is good (indicating 1)
+        bid = bidfactory.createBid(CONSTANTS.STATUS.GOOD, bidObj);
+        bid.bidderCode = bidCode;
+        bid.cpm = responseCPM;
+        bid.adUrl = jptResponseObj.content_url;
+        bid.width = jptResponseObj.width;
+        bid.height = jptResponseObj.height;
+
+        // Styles params
+        let styles = utils.getBidIdParameter('style', bidObj.params);
+        let stylesParams = {};
+        for (let currentValue in styles) {
+          if (styles.hasOwnProperty(currentValue)) {
+            let currentStyle = styles[currentValue];
+            for (let field in currentStyle) {
+              if (currentStyle.hasOwnProperty(field)) {
+                let styleField = styleParamsToFieldsMap[currentValue + '.' + field];
+                if (styleField !== undefined) {
+                  stylesParams[styleField] = currentStyle[field];
+                }
+              }
+            }
+          }
+        }
+        bid.adUrl += '&' + utils.parseQueryStringParameters(stylesParams);
+
+        // Custom params
+        let customParams = utils.getBidIdParameter('customParams', bidObj.params);
+        let customParamsArray = {};
+        for (let customField in customParams) {
+          if (customParams.hasOwnProperty(customField)) {
+            customParamsArray['c.' + customField] = customParams[customField];
+          }
+        }
+        let customParamsLink = utils.parseQueryStringParameters(customParamsArray);
+        if (customParamsLink) {
+          // Don't append a "&" here, we have already done it in parseQueryStringParameters
+          bid.adUrl += customParamsLink;
+        }
+
+        // Remove the trailing "&"
+        bid.adUrl = removeTrailingAmp(bid.adUrl);
+
+        bidmanager.addBidResponse(placementCode, bid);
+      } else {
+        // No response data
+        // @if NODE_ENV='debug'
+        utils.logMessage('No prebid response from Orbitsoft for placement code ' + placementCode);
+        // @endif
+        // indicate that there is no bid for this placement
+        bid = bidfactory.createBid(CONSTANTS.STATUS.NO_BID, bidObj);
+        bid.bidderCode = bidCode;
+        bidmanager.addBidResponse(placementCode, bid);
+      }
+    } else {
+      // No response data
+      // @if NODE_ENV='debug'
+      utils.logMessage('No prebid response for placement');
+      // @endif
+    }
+  };
+
+  return Object.assign(this, {
+    callBids: baseAdapter.callBids,
+    setBidderCode: baseAdapter.setBidderCode,
+    buildJPTCall: buildJPTCall
+  });
+};
+
+adaptermanager.registerBidAdapter(new OrbitsoftAdapter(), ORBITSOFT_BIDDERCODE);
+adaptermanager.aliasBidAdapter('orbitsoft', 'orbitadserving');
```

Comment:
why so many aliases?

Reply:
We rebranded recently from Orbitscripts with Orbit Ad Serving product to Orbitsoft.
If too much, I can delete.

Reply:
I think it is a bit confusing to have so many. Typically if the code changes we put the alias in to help with transition but since this is a new adapter I don't think you need them. You should use your current preferred name only.

Reply:
@mkendall07 It is fair remark. Deleted it.